src/Policy/AndPolicy.php

```diff
@@ -34,4 +36,25 @@ public function isCompliant(NormalizerContextInterface $context, $object): bool
 
         return true;
     }
+
+    public function isCompliantIncludingPath(object $object, NormalizerContextInterface $context, string $path): bool
```

Comment:
```suggestion
    public function isCompliantIncludingPath(string $path, object $object, NormalizerContextInterface $context): bool
```

Reply:
the funny thing is on the normalizer it's still a different parameter order

Reply:
https://github.com/chubbyphp/chubbyphp-serialization/blob/master/src/Normalizer/FieldNormalizerInterface.php#L18-L23

Reply:
i don't see the difference

Reply:
https://github.com/chubbyphp/chubbyphp-serialization/blob/e68c7f616f2a9e38fc35a1a62d727a9fa181713a/src/Normalizer/NormalizerInterface.php#L18

Reply:
@healerz  @marko-ilic thats not the same, the FieldNormalizerInterface is the relevant here, normalize on Normalizer has a diffrent signature cause default handling

Reply:
@dominikzogg yes I'm aware that FieldNormalizer != Normalizer, nevertheless they have the same parameters, but in a different order ;) Just wanted to mention it, all good 👍 Initially, for some reason we thought it's the other way around (Signature of `Normalizer::normalize` <-> `FieldNormalizer::normalizeField`)

Reply:
and good catch btw :)